src/AggregateSpecsPlugin.ts

```diff
@@ -72,10 +80,9 @@ export const PgAggregatesSpecsPlugin: GraphileConfig.Plugin = {
             id: "sum",
             humanLabel: "sum",
             HumanLabel: "Sum",
-            isSuitableType: isNumberLike,
+            isSuitableType: isIntervalLike || isNumberLike,
```

Comment:
```
            isSuitableType: isIntervalLike || isNumberLike,
```

is equivalent to

```
            isSuitableType: isIntervalLike,
```

since `isIntervalLike` (a function) is truthy. Which is presumably why you had to define `isIntervalLike` as true if the input is number like...